studio/components/grid/components/header/Header.tsx

```diff
@@ -349,14 +349,18 @@ const RowHeader = ({ table, sorts, filters }: RowHeaderProps) => {
     const rows = allRowsSelected
       ? await state.rowService!.fetchAllData(filters, sorts)
       : allRows.filter((x) => selectedRows.has(x.idx))
-    const formattedRows = rows.map((row) => {
-      const formattedRow = row
-      Object.keys(row).map((column) => {
-        if (typeof row[column] === 'object' && row[column] !== null)
-          formattedRow[column] = JSON.stringify(formattedRow[column])
-      })
-      return formattedRow
-    })
+    const formattedRows = useMemo(() => {
+      return rows.map((row) => {
+        const formattedRow = { ...row };
+        Object.keys(row).forEach((column) => {
+          if (typeof row[column] === 'object' && row[column] !== null) {
+            formattedRow[column] = JSON.stringify(formattedRow[column]);
+          }
+        });
+        return formattedRow;
+      });
+    }, [rows]);
```

Comment:
This `useMemo` is unnecessary because it's inside a function and causing the build to fail

Reply:
> This `useMemo` is unnecessary because it's inside a function and causing the build to fail

Oops! Sorry for it